src/js/controllers/tx-details.js

```diff
@@ -36,6 +54,69 @@ angular.module('copayApp.controllers').controller('txDetailsController', functio
     });
   });
 
+  function setLoading(name, state) {
+    $scope.loadingService[name] = state;
+    $timeout(function() {
+      $scope.$apply();
+    });
+  };
+
+  function verifyByThirdParties() {
+    if (lodash.isEmpty($scope.availableServices) || $scope.wallet.network == 'testnet') return;
+
+    lodash.each($scope.availableServices, function(service) {
+      setLoading(service.name, true);
+
+      verifyByThirdPartyService.getTx(service, txId, function(err, params) {
+        if (err) {
+          setLoading(service.name, false);
+          $scope.serviceError[service.name] = true;
+          $log.warn('Could not get tx params from: ' + service.name);
+          checkError();
+        } else
+          compareAndUpdateParams(params);
+
+        serviceCounter += 1;
+      });
+    });
+  };
+
+  function compareAndUpdateParams(params) {
```

Comment:
inside the service.

Please do not use `params`. Parameters are a function input.  It is a generic name.